.circleci/scripts/windows_cuda_install.sh

```diff
@@ -16,7 +14,7 @@ else
     exit 1
 fi
 
-if [[ "${CUDA_VERSION}" != "10" && "${JOB_EXECUTOR}" == "windows-with-nvidia-gpu" ]]; then
+if [[ "$CUDA_VERSION" =~ ^10.* && "${JOB_EXECUTOR}" == "windows-with-nvidia-gpu" ]]; then
```

Comment:
This introduced runtime failures in CI, because it should be `"$CUDA_VERSION" =~ ^11.*`

Reply:
Thank your help. So sorry for that.